app/model/block.py

```diff
@@ -10,11 +10,73 @@ def __init__(self):
         self.container = None
         self.questionnaire = None
         self.validation = None
-        self.questionniare = None
+        self.questionnaire = None
         self.templatable_properties = []
         self.display = Display()
 
     def add_section(self, section):
         if section not in self.sections:
             self.sections.append(section)
             section.container = self
+
+    def to_json(self):
+        json_dict = {
+            "id": self.id,
+            "title": self.title,
+            "sections": [],
+            "validation": [],
+            "display": {}
+        }
+
+        for section in self.sections:
+            json_dict['sections'].append(section.to_json())
+
+        if self.validation is not None:
+            for validation in self.validation:
+                json_dict['validation'].append(validation.to_json())
+
+        if self.display is not None:
+            json_dict['display'] = self.display.to_json()
+
+        return json_dict
+
+    def __eq__(self, other):
```

Comment:
With regard to doing a comparision at json string level, that approach didn't work as the order in which the properties appeared in the json seemed to vary